rest/changesttest/changes_collection_test.go

```diff
@@ -389,11 +390,11 @@ func TestMultiCollectionChangesCustomSyncFunctions(t *testing.T) {
 		},
 	}
 
-	rt, keyspaces := rest.NewRestTesterMultipleCollections(t, rtConfig, numCollections)
+	rt, _ := rest.NewRestTesterMultipleCollections(t, rtConfig, numCollections)
 	defer rt.Close()
 
-	c1Keyspace := keyspaces[0]
-	c2Keyspace := keyspaces[1]
+	c1Keyspace := strings.Join([]string{rt.GetDatabase().Name, dataStoreNames[0].ScopeName(), dataStoreNames[0].CollectionName()}, base.ScopeCollectionSeparator)
```

Comment:
We definitely don't want to have to be writing this for every test that wants to use keyspaces - that was the entire motivation to return the keyspace as a simple string. What required this change?

Reply:
Refactoring mishap.